.circleci/lint.sh

```diff
@@ -18,23 +18,18 @@ MODULE="$(basename -s .info.yml -- ./*.info.yml)"
 
 #-------------------------------------------------------------------------------
 
-echo "==> Lint code for module $MODULE."
```

Comment:
please review the old code: it was all running in the `build` directory to keep the module's (root) `composer.json` without any dependencies. this changes everything to run in the root of the project which is not what we want.

please revert to make it all run from the `build` directory. note that `build.sh` already adds all required development dependencies to the `build/composer.json` file

Reply:
I have reverted root composer.json and make it run all from build.sh